entries/r/recruitee.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Recruitee": {
+    "domain": "recruitee.com",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support.recruitee.com/en/articles/5209837-protect-your-account-with-two-factor-authentication-2fa",
```

```suggestion
    "documentation": "https://support.recruitee.com/en/articles/5209837",
```
This URL can be shortened